src/Google.Api.Gax/CallSettings.cs

```diff
@@ -62,6 +73,65 @@ public sealed class CallSettings
             WriteOptions = WriteOptions,
             PropagationToken = PropagationToken,
             Credentials = Credentials,
+            RetrySettings = RetrySettings?.Clone(),
         };
+
+        // Merge other into this.
```

Comment:
Will we always have one of the two as non-null? I was wondering whether a static method might be more flexible.
Also, I'd expected (in my head) for Merge to return a new one instead of merging it into an existing one. I suspect that would allow less defensive cloning - but I admit I haven't looked through carefully. Probably something for another PR after pondering.